whitelist.txt

```diff
@@ -1,6 +1,15 @@
 # This is a whitelist of allowed words for flake8-spellcheck plugin.
 # IMPORTANT: please keep this list alphabetically sorted by whitelisted words
 
+#SQLAlchemy parameter name
+backref
```

Comment:
After we move relationship from `User` model to `Topic` model we will get rid of `backref` parameter, therefore we can also remove it from this list. Just a reminder to not forgot